quipucords/scanner/vcenter/inspect.py

```diff
@@ -204,6 +204,7 @@ def parse_vm_props(self, props, host_dict):
                 facts['vm.host.cpu_cores'] = host_facts['host.cpu_cores']
                 facts['vm.host.cpu_count'] = host_facts['host.cpu_count']
                 facts['vm.host.cpu_threads'] = host_facts['host.cpu_threads']
+                facts['vm.host.cpu_count'] = host_facts['host.cpu_count']
```

Comment:
This should already have been set here [:point_up:](https://github.com/quipucords/quipucords/pull/1363/commits/f67168d2ea5442f8e3fd3227ab235b54cfbd1fa2#diff-6da9bab18f203288f5a6c1e1e6000303R205)